docker/devenv/files/start-tmux.sh

```diff
@@ -26,6 +26,7 @@ tmux send-keys -t penpot 'yarn run watch' enter
 tmux new-window -t penpot:1 -n 'frontend shadow'
 tmux select-window -t penpot:1
 tmux send-keys -t penpot 'cd penpot/frontend' enter C-l
+tmux send-keys -t penpot 'yarn run build:renderer' enter C-l
```

Comment:
I think for now, we don't need to add new window to the tmux just for building process. This is still an experiment and it should be build on demand (In the same way this renderer is under configuration flag, so we activate it explicitly when we going to test it, we can open a specific window on demand for building it)

Adding new window for all users independently if they go or not to develop/test this I think is not necessary